tests/discovery/test_discovery.py

```
@@ -40,19 +76,41 @@
+# Wrap direct cases into signposted cases
 SIGNPOSTED_CASES = list()
 for uri, mime, length in DIRECT_CASES:
     link = wrap_signpost_uri(uri)
     SIGNPOSTED_CASES.append((link, mime, length))
 
-ALL_CASES = DIRECT_CASES + SIGNPOSTED_CASES
+ALL_CASES = REMOTE_CASES + SIGNPOSTED_CASES
+
+
+class CustomHTTPRequestHandler(SimpleHTTPRequestHandler):
+    def guess_type(self, path):
```

Ruff

Missing return type annotation for public function `guess_type`

(ANN201)